readthedocs/api/v2/views/footer_views.py

```diff
@@ -147,7 +148,7 @@ def _get_context(self):
         page_slug = self.request.GET.get('page', '')
         path = ''
         if page_slug and page_slug != 'index':
-            if version.documentation_type == 'sphinx_htmldir':
+            if version.documentation_type in {SPHINX_HTMLDIR, MKDOCS}:
```

Comment:
I vaguely remember there is an option somewhere whether to use the `.html` suffix or not. I scanned the MkDocs documentation but I can't find it anymore. Or was it a RTD flag?

Reply:
yes, there is, we already keep track of that

https://github.com/readthedocs/readthedocs.org/blob/master/readthedocs/projects/constants.py#L17-L17

Reply:
Oh, thanks. By backtracking the use of that constant I eventually found it: https://www.mkdocs.org/user-guide/configuration/#use_directory_urls